### PR TITLE
[SRVCOM-1829] Update abstracts and structure for admin console topics

### PR DESCRIPTION
**Pull request by abrennan89**
Version(s):
OCP 4.6+ (maybe needs manual cherry picks for versions older than 4.9 since there was no eventing in admin console pre-4.7, and no OSD docs pre-4.9)

Issue:
https://issues.redhat.com/browse/SRVCOM-1829

Link to docs preview:
- http://file.rdu.redhat.com/abrennan/030622/SRVCOM-1829/serverless/admin_guide/serverless-admin-perspective.html

### CODE REVIEW

**Review by pierDipi — APPROVED** [review]
LGTM

File: modules/serverless-creating-channel-admin-web-console.adoc [hunk 1]
```diff
@@ -6,18 +6,14 @@
...
 = Creating a channel by using the Administrator perspective
 
-ifdef::openshift-enterprise[]
-If you have cluster administrator permissions, you can create a channel by using the *Administrator* perspective in the web console.
-endif::[]
-
-ifdef::openshift-dedicated[]
-If you have cluster or dedicated administrator permissions, you can create a channel by using the *Administrator* perspective in the web console.
-endif::[]
+include::snippets/serverless-channels-intro.adoc[]
```
Comment (skonto) [inline modules/serverless-creating-channel-admin-web-console.adoc:9]: I guess this is the same for the develop view right?

Comment (abrennan89) [reply modules/serverless-creating-channel-admin-web-console.adoc:9]: Yeah, it's just content reuse anywhere there's an "include" statement, you should be able to look at both in the preview link by navigating around the docs 🙂

**Comment by skonto** [comment]
LGTM

**Comment by jrangelramos** [comment]
QE Approved

**Comment by opayne1** [comment]
lgtm!

**Comment by mburke5678** [comment]
@abrennan89 One minor nit. Otherwise LGTM.

**Pull request merged**